lib/charms/prometheus_k8s/v1/prometheus.py

```diff
@@ -279,30 +231,79 @@ def jobs(self):
             its scrape targets.
         """
         scrape_jobs = []
-        for job in self._stored.jobs.values():
-            scrape_jobs.append(json.loads(job))
+
+        for relation in self._charm.model.relations[self._relation_name]:
+            job = self._static_scrape_config(relation)
+            if job:
+                scrape_jobs.append(job)
 
         return scrape_jobs
 
+    def _static_scrape_config(self, relation):
+        """Generate the static scrape configuration for a relation.
```

Comment:
consider documenting how the full scrape_config looks like ("pseudo schema"?)

Reply:
Done.